decidim-assemblies/app/types/decidim/assemblies/assembly_type.rb

```diff
@@ -0,0 +1,62 @@
+      field :participatoryScope, Decidim::Core::TranslatedFieldType, "What is decided on this assembly", property: :participatory_scope
+      field :participatoryStructure, Decidim::Core::TranslatedFieldType, "How it is decided on this assembly", property: :participatory_structure
+      field :showStatistics, types.Boolean, "If this assembly should show statistics", property: :show_statistics
+      field :scopesEnabled, types.Boolean, "If this assembly has scopes enabled", property: :scopes_enabled
+      field :privateSpace, types.Boolean, "If this assembly is a private space", property: :private_space
+      field :area, Decidim::Core::AreaApiType, "Area of this assembly", property: :area
+      field :parent, Decidim::Assemblies::AssemblyType, "The parent assembly of this assembly", property: :parent
+      field :parentsPath, types.String, "Assembly hierarchy representation", property: :parents_path
+      field :childrenCount, types.Int, "Number of children assemblies", property: :children_count
+      field :purposeOfAction, Decidim::Core::TranslatedFieldType, "Purpose of action", property: :purpose_of_action
+      field :composition, Decidim::Core::TranslatedFieldType, "Composition of this assembly"
+      field :assemblyType, types.String, "Type of the assembly", property: :assembly_type
+      field :creationDate, Decidim::Core::DateType, "Creation date of this assembly", property: :creation_date
+      field :createdByOther, Decidim::Core::TranslatedFieldType, "Created by other", property: :created_by_other
+      field :duration, Decidim::Core::DateType, "Duration of this assembly"
+      field :includedAt, Decidim::Core::DateType, "Included at", property: :included_at
+      field :closingDate, Decidim::Core::DateType, "Closing date of the assembly", property: :closing_date
+      field :closingDateReason, Decidim::Core::TranslatedFieldType, "Closing date reason of this assembly", property: :closing_date_reason
+      field :internalOrganisation, Decidim::Core::TranslatedFieldType, "Internal organisation of this assembly", property: :internal_organisation
+      field :isTransparent, types.Boolean, "If this assembly is transparent", property: :is_transparent
+      field :specialFeatures, Decidim::Core::TranslatedFieldType, "Special features of this assembly", property: :special_features
+      field :twitterHandler, types.String, "Twitter handler", property: :twitter_handler
+      field :instagramHandler, types.String, "Instagram handler", property: :instagram_handler
+      field :facebookHandler, types.String, "Facebook handler", property: :facebook_handler
+      field :youtubeHandler, types.String, "Youtube handler", property: :youtube_handler
+      field :githubHandler, types.String, "Github handler", property: :github_handler
+
+      field :members, types[Decidim::Assemblies::AssemblyMemberType], "Members of this assembly"
+      field :childrens, types[Decidim::Assemblies::AssemblyType], "Childrens of this assembly"
```

Comment:
```suggestion
      field :childrens, !types[Decidim::Assemblies::AssemblyType], "Childrens of this assembly"
```
Not sure, just to avoid nulls